### ISSUE TITLE
[1.3-cordova-beta.5] Show splash screen and crash immediately

### ISSUE THREAD

**Issue by chip** [meteor/meteor#6513]
I have a bare bones `ios` app that **shows the splash screen and then crashes immediately**.  

I'm running the following:
* Meteor release `1.3-cordova-beta.5`
* iOS `9.2.1`
* Xcode `7.2.1`

I've created a repo to reproduce the error: https://github.com/chip/ios-crash

Here are the steps to reproduce:

### Install
```
git clone git@github.com:chip/ios-crash.git
cd ios-crash                                           
rm -rf .meteor/local/cordova-build
meteor build ../ios-crash-build --server http://localhost:3000
open ../ios-crash-build/ios/project/ios-crash.xcodeproj
```

### Open Xcode

1. Under the **General** tab of the project, set **Deployment target** to `9.2`. (I've also tested this with `8.0`)
2. Set **Devices** to `Universal`
3. Select **Archive** from the **Product** menu and follow all of the steps to create an `.ipa` file.
4. Double click the `.ipa` to install on device and perform syncing.
5. Launch app and watch the splash screen show briefly before crashing

### Side notes

I had this behavior in another more complex app and eventually found that this was originally reported by @jhatchr in #6050, so that's why I tried to reproduce it with something simple. I've experienced it with both `1.3-cordova-beta.5` and `1.3-rc.2` releases.

Xcode completely baffles me, so it's possibly something I'm missing in the build settings, too. However, I have had no problems creating builds using release `1.2.1`, but that release didn't have HCP (Hot Code Push), so that's why I'm here.

Thanks in advance for any suggestions, and I really appreciate all of the hard work that is being done on this fantastic platform!

**Comment by martijnwalraven** [comment]
Thanks for writing such an exemplary issue: well written, researched, and with a reproduction!

I believe this is most likely a duplicate of #6492, which turned out to be a bug in Cordova iOS 4.0. A workaround should be in the next RC.

In the meantime, could you try manually adding `@executable_path/Frameworks` to the `Runpath search paths` for `release` to see if this fixes your issue?

**Comment by chip** [comment]
@martijnwalraven - Thanks so much for your prompt reply. I had followed that issue and actually tried it on my complex app, but it didn't work. Then last night I tried again using my bare bones app and **it worked** on my MacBook Air. However, this morning I tried the same procedure for both apps on my iMac and it still has the same issue.

I have tried following Apple's docs as closely as possible, but have yet to find an approach that consistently works. My hunch is it has something to do with the **profiles**, but I'm not certain on that. I will keep trying and follow up here as I have more details. Thanks again for your help.

**Comment by chip** [comment]
@martijnwalraven:

## TLDR;

Finally got it!

Now that I have this documented, **I would like to experiment with getting a proper `config.xml` that I can reuse**. I plan on experimenting with using the **cordova-build-override** project root directory, but see the  link from the Meteor docs just goes to a Wiki page with no details. Do the docs and Wiki need to be updated or is this feature no longer available? 

## FULL DETAILS

I tried so many things, and I realize some of the items in this list might not be necessary, but I wanted to document it here in case it helps someone else. Here are the highlights of what I did:

### Command line

* **Remove all Provisioning profiles on my local machine** to start with a clean slate:

`rm ~/Library/MobileDevice/Provisioning\ Profiles/*`

### Xcode

* From within Xcode, Select **File => Preferences => Accounts**, click on the proper Team from under the Team Name section and click “View Details”.

* Click “Download All” to fetch the profiles from Apple Developer Membership site

### Xcode Build Settings

* I applied your solution above. Per @martijnwalraven: *In the meantime, could you try manually adding @executable_path/Frameworks to the Runpath search paths for release to see if this fixes your issue?* 

Be sure to do this for the Build Settings under **BOTH the Project and Targets**. I found this after getting the Cordova error https://issues.apache.org/jira/browse/CB-10885 when using the Simulator and suddenly discovered that the settings for BOTH Project and Target were not exactly the same.

* **Set Provision profile to “Automatic” under BOTH Project and Targets**. I think I originally missed these because the Projects and Targets lists were hidden

### Project => Archive from the menu

* **Code signing identity** under **Build Settings** should be `iOS Developer`.

* I **COULD NOT** export for `Ad Hoc development`. This is in direct conflict with what Apple suggests at https://developer.apple.com/library/ios/qa/qa1814/_index.html. I **COULD** export the file as `Save for  Development Deployment` so that it picks the proper provision.

Thanks for your help! :+1:
